src/components/icons/StarIcon.tsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
-import { createIcon } from '@chakra-ui/react';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+type StarProps = IconProps & {
+  fill?: 'none';
+  stroke?: 'currentColor';
```

Comment:
extending props as these values are not supported and we need them for animations in listings